### PR TITLE
Add gRPC interceptors setting in configuration

### LINKED ISSUE #207: Allow developers write their own interceptors
The current PHP package has a hidden feature - interceptors for grpc requests. 
As seen here: `\Google\Ads\GoogleAds\Lib\V1\ServiceClientFactoryTrait::getGoogleAdsClientOptions`
One application for them is logging (which is already implemented).

For a regular developer, it is not easy to use them and there is no documentation for it available. I feel there is a HUGE benefit in making them available to the general public. For example, I used them to create a VCR layer which would record and playback of all requests and responses (for purposes of testing).

The problem is that `\Google\Ads\GoogleAds\Lib\V1\GoogleAdsClient` is final and doesn't let you customize interceptors.

What do you think?

### PR DESCRIPTION
**Pull request by PierrickVoulet**
Fixes #207

### CODE REVIEW

**Comment by fiboknacky** [comment]
This will be probably reviewed after I release support for v9. Could you please wait until that and patch to support v9 too? Thanks.

**Comment by PierrickVoulet** [comment]
> This will be probably reviewed after I release support for v9. Could you please wait until that and patch to support v9 too? Thanks.

Done. Feel free to take a loot whenever you can.

File: src/Google/Ads/GoogleAds/Lib/ConfigurationTrait.php [hunk 1]
```diff
@@ -35,6 +36,7 @@ trait ConfigurationTrait
     private $logLevel;
     private $proxy;
     private $transport;
+    private $grpcInterceptors;
```
Comment (fiboknacky) [inline src/Google/Ads/GoogleAds/Lib/ConfigurationTrait.php:39]: Same question with the other PR. How do you plan to support this in the config file?

Comment (PierrickVoulet) [reply src/Google/Ads/GoogleAds/Lib/ConfigurationTrait.php:39]: There is no plans to support this in the config file.

We could do it in the future if we decide to support config file in `PHP` format to complement or replace the current `INI` format but this is out of scope of this PR.

Comment (fiboknacky) [reply src/Google/Ads/GoogleAds/Lib/ConfigurationTrait.php:39]: Ditto. Please see the comment in the other PR

**Review by fiboknacky — APPROVED** [review]

**Comment by lezhnev74** [comment]
Awesome!

**Pull request merged**